docs/book/src/reference/versions.md

```diff
@@ -63,77 +63,81 @@ These diagrams show the relationships between components in a Cluster API releas
-| Kubernetes v1.21 + etcd/v3 | ✓                    | ✓                    | ✓                   | ✓                        |
-| Kubernetes v1.22 + etcd/v3 | ✓* (only workload)   | ✓                    | ✓                   | ✓                        |
-| Kubernetes v1.23 + etcd/v3 |                      | ✓*                   | ✓*                  | ✓                        |
-| Kubernetes v1.24 + etcd/v3 |                      |                      |                     | ✓                        |
+|                            |  v0.3 (v1alpha3)     |  v0.4 (v1alpha4)     |  v1.0 (v1beta1)     |  v1.1 (v1beta1)    |  v1.2 (v1beta1)     |  v1.3 (v1beta1)     |
+|----------------------------|----------------------|----------------------|---------------------|--------------------|---------------------|---------------------|
+| Kubernetes v1.16 + etcd/v3 | ✓                    |                      |                     |                    |                     |                     |
+| Kubernetes v1.17 + etcd/v3 | ✓                    |                      |                     |                    |                     |                     |
+| Kubernetes v1.18 + etcd/v3 | ✓                    | ✓ (only workload)    | ✓ (only workload)   | ✓ (only workload)  | ✓ (only workload)   | ✓ (only workload)   |
+| Kubernetes v1.19 + etcd/v3 | ✓                    | ✓                    | ✓                   | ✓                  | ✓ (only workload)   | ✓ (only workload)   |
+| Kubernetes v1.20 + etcd/v3 | ✓                    | ✓                    | ✓                   | ✓                  | ✓                   | ✓                   |
+| Kubernetes v1.21 + etcd/v3 | ✓                    | ✓                    | ✓                   | ✓                  | ✓                   | ✓                   |
+| Kubernetes v1.22 + etcd/v3 | ✓* (only workload)   | ✓                    | ✓                   | ✓                  | ✓                   | ✓                   |
+| Kubernetes v1.23 + etcd/v3 |                      | ✓*                   | ✓*                  | ✓                  | ✓                   | ✓                   |
+| Kubernetes v1.24 + etcd/v3 |                      |                      |                     | ✓                  | ✓                   | ✓                   |
 
 The Kubeadm Control Plane Provider talks to the API server and etcd members of every Workload Cluster whose control plane it owns. It uses the etcd v3 API.
 
 The Kubeadm Control Plane requires the Kubeadm Bootstrap Provider.
 
 \*  Newer versions of CoreDNS may not be compatible as an upgrade target for clusters managed with Cluster API. Kubernetes versions marked on the table are supported as an upgrade target only if CoreDNS is not upgraded to the latest version supported by the respective Kubernetes version. The versions supported are represented in the below table.
 
+#### CoreDNS
```

Comment:
nit: This should be a sub section of KCP. Maybe let's use: **CoreDNS**: ? So it's clearly a subsection not sure if 5 #'s is distinguishable :)